ote_cli/ote_cli/tools/train.py

```diff
@@ -93,7 +94,6 @@ def parse_args():
     )
     parser.add_argument(
         "--hpo-time-ratio",
-        default=4,
```

Comment:
Why the default value is removed here?

Reply:
Because default parameter enables that option and if bool option '--enable-hpo' isn't enabled at the same call, that cause exception from line 122 "Parameter --hpo-time-ratio must be used with --enable-hpo key"